mongo_seed/seed_data.sh

```diff
@@ -3,4 +3,6 @@
 
 mongoimport --host clearlydefined_mongo_db --db clearlydefined --collection definitions-trimmed --type json --file definitions/angular.json --jsonArray
 mongoimport --host clearlydefined_mongo_db --db clearlydefined --collection definitions-trimmed --type json --file definitions/flyway-maven-plugin.json --jsonArray
+mongoimport --host clearlydefined_mongo_db --db clearlydefined --collection definitions-paged --type json --file definitions/angular-paged.json --jsonArray
+mongoimport --host clearlydefined_mongo_db --db clearlydefined --collection definitions-paged --type json --file definitions/flyway-maven-plugin-paged.json --jsonArray
```

Comment:
If approved, "clearlydefined/docker_dev_env_experiment_clearlydefined_mongo_seed" docker image needs to be updated.